extras/kinesis-asl/src/main/scala/org/apache/spark/streaming/kinesis/KinesisUtils.scala

```diff
@@ -90,8 +93,9 @@ object KinesisUtils {
       endpointUrl: String, 
       checkpointInterval: Duration,
       initialPositionInStream: InitialPositionInStream,
-      storageLevel: StorageLevel): JavaReceiverInputDStream[Array[Byte]] = {
+      storageLevel: StorageLevel,
```

Comment:
Yes, the idea is not to change the API. I suppose this is less important as a module, but still. I would have thought MiMa would flag this but I don't know that it checks this module.

I think this is really something for @cfregly et al to review